addons/point_of_sale/static/src/js/models.js

```diff
@@ -1180,7 +1180,7 @@ exports.load_fields = function(model_name, fields) {
 //  context: [Dict|function] the openerp context for the model read
 //  condition: [function] do not load the models if it evaluates to
 //             false.
-//  loaded: [function(self,model)] this function is called once the 
+//  loaded: [function(self,model)] this function is called once the
```

Comment:
Do not change anything not related to the change itself please (that will help with noisy changes).

Reply:
An oversight, sorry. I didn't take my sublime, yet. Will rebase and fix shortly...